### DISCUSSION TITLE
Having issues with seeing failing tests' output

### DISCUSSION THREAD

**Discussion by zarembas** [community/community#68236] [Actions]
### Select Topic Area

Bug

### Body

This started to happen a lot more often on the repo I'm working on:

For example: https://github.com/DFE-Digital/apply-for-teacher-training/actions/runs/6298202662/job/17096705382?pr=8588

When trying to expand a step from the build job, more often than not there will be no output or partial output and this makes it impossible to see the failed tests.

These are the errors I'm getting in the console when expanding the output:

```
RangeError: Maximum call stack size exceeded
https://api.github.com/_private/browser/stats 500 (Internal Server Error)
```

Can anyone help? The build takes quite a bit of time to run and not seeing what tests fail at the end is far from ideal.

**Screenshots:**
<img width="1097" alt="Screenshot 2023-09-25 at 16 13 00" src="https://github.com/community/community/assets/1636476/18316b67-4bc5-4f8f-a70b-edbd5e479ed5">

<img width="663" alt="Screenshot 2023-09-25 at 16 13 10" src="https://github.com/community/community/assets/1636476/6c6f1732-593c-492d-9761-990350088e7a">

**Comment by tomas-stefano** [comment]
Does anyone has any updates on this?

**Comment by tomas-stefano** [comment]
Hey, does anyone has any updates on this?
